lib/beaker/dsl/install_utils/foss_utils.rb

```diff
@@ -890,7 +895,7 @@ def install_repo_configs(host, buildserver_url, package_name, build_version, cop
                                   repo_filename,
                                   copy_dir )
 
-          if host[:platform] =~ /cisco-5/
+          if host[:platform] =~ /cisco-nexus/
             to_path = "#{host.package_config_dir}/#{File.basename(repo)}"
```

Comment:
Can you add a comment explaining what this is doing on NX-OS and why? Are you confident that it doesn't apply to IOS XR?